operator/cmd/mesh/manifest-apply.go

```diff
@@ -59,6 +67,7 @@ func addManifestApplyFlags(cmd *cobra.Command, args *manifestApplyArgs) {
 	cmd.PersistentFlags().BoolVarP(&args.wait, "wait", "w", false, "Wait, if set will wait until all Pods, Services, and minimum number of Pods "+
 		"of a Deployment are in a ready state before the command exits. It will wait for a maximum duration of --readiness-timeout seconds")
 	cmd.PersistentFlags().StringArrayVarP(&args.set, "set", "s", nil, SetFlagHelpStr)
+	cmd.PersistentFlags().StringVarP(&args.certDir, "cert-dir", "t", "", "The path where to save and store the certificates")
```

Comment:
Which certificates ? This is too vague - please qualify ( private key and CA certificates ?)"

Reply:
Also the flag name may be more specific - cert-dir is typically used for the workload certs.

Reply:
How about "root-cert" ? ( without dir - so later we can expand this a bit, maybe a zip file, or 
device)